pyopia/instrument/silcam.py

```diff
@@ -51,6 +51,90 @@ def load_mono8(filename):
     return img
 
 
+def load_brg8(filename):
```

Comment:
Suggest calling this function load_bayer_rgb8() so as not to confuse b with blue